crates/telio-wg/src/wg.rs

```diff
@@ -471,7 +471,7 @@ impl State {
 
     async fn uapi_request(&mut self, cmd: &Cmd) -> Result<Response, Error> {
         let ret = self.adapter.send_uapi_cmd(cmd).await?;
-        telio_log_debug!("UAPI request: {}, response: {}", &cmd.to_string(), &ret);
+        telio_log_trace!("UAPI request: {}, response: {:?}", &cmd.to_string(), &ret);
```

Comment:
maybe `telio_log_debug!()` would be better, as this would mean, that builds provided to app teams will have these compiled in, and may help debugging. 

`telio_log_trace!()` is not compiled-in in the above scenario

Reply:
This is printed every second though it was too much spam, but I understand your POV, will change.

Reply:
Change back to DEBUG log_level.